### PR TITLE
Update zones.json

### PR DESCRIPTION
**Pull request by gopikrishna1793**
Updated installed capacity for India as of May 22.

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -2012,21 +2012,34 @@
...
+      "gas": 24879.21,
+      "hydro": 46722.52,
       "nuclear": 6780,
-      "oil": 509.7,
+      "oil": 509.71,
       "solar": 42335.49,
       "wind": 39486.65
     },
-    "contributors": ["systemcatch"],
+    "comment": "Installed capacity as on May-2022 https://npp.gov.in/public-reports/cea/monthly/installcap/2022/MAY/capacity2-Southern-2022-05.pdf",
```
Comment (VIKTORVAV99) [inline config/zones.json:2023]: This source should be located in DATA_SOURCES.md instead.

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Overall this looks good, just move the source into `DATA_SOURCES.md` and this should be ready to merge!

Thanks for opening this PR 🙂

**Comment by VIKTORVAV99** [comment]
@gopikrishna1793 I'm sorry if there was a misunderstanding but you didn't have to close this PR.

The reason we don't put sources in the `zones.json` file is to avoid making the front-end bundle unnecessary big as well as to have all sources in one place.

So feel free to reopen this PR, then just move the source into `DATA_SOURCES.md` and we can get this merged.

**Comment by gopikrishna1793** [comment]
@VIKTORVAV99 Actually I'm new to GitHub. I thought the task was completed that's the reason I closed. I will do PR again.

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99 Actually I'm new to GitHub. I thought the task was completed that's the reason I closed. I will do PR again.

No worries! I'll help you if you got any questions.

For now it would be great if you could edit this PR and move the data source to the other file. If you need any help with it just let me know and I can assist.

**Comment by VIKTORVAV99** [comment]
@gopikrishna1793 I see you opened a new PR at #4292 and that is fine this time but try and keep all changes in one PR in the future. 👍🏻 

Could you remove the comments in the `zones.json` file you added in this PR?

After that I will review the all the values, I saw some thing that might need a change already but I will help with that. 🙂

**Comment by gopikrishna1793** [comment]
@VIKTORVAV99 Sure will keep all changes in one PR. Could you help me with how to edit zones.json so that I can remove comments?

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99 Sure will keep all changes in one PR. Could you help me with how to edit zones.json so that I can remove comments?

Sure thing!
We only need to remove the comments since it looks like the updated source in the other PR is already in the data sources file.
I assume you are using the github web interface.
Steps:
1. Go to changed files in this PR.
2. Press the `...` button on the file you want to edit. (Should be on the far right side of the filename.
3. Press edit file.
4. Do all the edits you want.
5. Save the file by pressing "commit changes" the file. (the default name, description and options should be fine this time.)

(In the future it might be easier to download vscode or use the web version at https://vscode.dev/ when editing multiple files.)

Once this is done I'll do another review and double check everything.

**Comment by gopikrishna1793** [comment]
@VIKTORVAV99 I'm using vscode editor and GitHub desktop. I removed comments from the zones.json file. On the Github desktop, I got the option to commit to master.

**Comment by VIKTORVAV99** [comment]
@gopikrishna1793 great, I saw the commit ended up on the other PR but that doesn't really matter this time.

I won't be able to review it right away but hopefully I can do it by the end of the day, otherwise I'll do it tomorrow.

**Comment by VIKTORVAV99** [comment]
I am going to close this PR in favor of #4290 as it has newer changes and has a ongoing review.

**Pull request closed without merging**